test/no-error-ctor-with-notthrows.js

```diff
@@ -0,0 +1,210 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/no-error-ctor-with-notthrows';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	},
+	parserOptions: {
+		ecmaVersion: 2019
+	}
+});
+
+const errors = [{ruleId: 'no-error-ctor-with-notthrows'}];
+
+const header = `const test = require('ava');\n`; // eslint-disable-line quotes
+
+ruleTester.run('no-error-ctor-with-notthrows', rule, {
+	valid: [
+		`${header}
+		test('some test',t => {
```

Comment:
```suggestion
		test('some test', t => {
```
Could you fix this for all the other times ?